model/src/test/resources/org/cloudfoundry/identity/uaa/oauth/jwk/JwkSet-Keycloak.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
This is great IMO. So much easier to read the value. Thanks!

Reply:
@strehle and I agree.

Reply:
@strehle and I agree.